app/webpacker/components/Results/Rankings/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { Container } from 'semantic-ui-react';
+import { useQuery } from '@tanstack/react-query';
+import RankingsTable from './RankingsTable';
+import WCAQueryClientProvider from '../../../lib/providers/WCAQueryClientProvider';
+import { getRankings } from '../api/rankings';
+import Loading from '../../Requests/Loading';
+import { rankingsUrl } from '../../../lib/requests/routes.js.erb';
+import ResultsFilter from '../resultsFilter';
+
+export default function Wrapper({
+  event, region, year, rankingType, gender, show,
+}) {
+  return (
+    <WCAQueryClientProvider>
+      <Rankings
+        initialEvent={event}
+        initialRegion={region}
+        initialYear={year}
+        initialRankingType={rankingType}
+        initialGender={gender}
+        initialShow={show}
+      />
+    </WCAQueryClientProvider>
+  );
+}
+
+export function Rankings({
+  initialEvent, initialRegion, initialRankingType, initialGender, initialShow,
+}) {
+  const [event, setEvent] = useState(initialEvent);
+  const [region, setRegion] = useState(initialRegion ?? 'all');
+  const [rankingType, setRankingType] = useState(initialRankingType);
+  const [gender, setGender] = useState(initialGender);
+  const [show, setShow] = useState(initialShow ?? 'Persons');
+
+  const filterState = useMemo(() => ({
```

Comment:
A reducer might be good here? Then only one setter/callback needs to be passed to the `ResultsFilter` component.
Setters aren't really 'state' so it feels off to include them in `filterState`.